executor/mock/standalone/standalone_test.go

```diff
@@ -847,3 +848,16 @@ func testTwoCPUs(t *testing.T, jobID string) {
 		t.Fail()
 	}
 }
+
+func testTty(t *testing.T, jobID string) {
+	ji := &mock.JobInput{
+		ImageName:     ubuntu.name,
+		Version:       ubuntu.tag,
+		EntrypointOld: `/usr/bin/tty`,
```

Comment:
Does this test to see if TTY is enabled (does it fail if TTY is disabled, if so, does it make sense to add a negative test?)

Reply:
Yeah - `tty` will fail if there's no tty present.  I can add a negative test.